src/components/modular-content/decision-tree/decision-tree.njk

```diff
@@ -65,6 +65,18 @@
                   {{ indexToLetter(index, true) }}:
                 </span>
                 {{ choice.content }}
+                <template v-if="choice.helpLink" >
+                  <br>
+                  <a
+                    @click.stop
+                    :href="choice.helpLink.externalUrl ? choice.helpLink.externalUrl : `/${ locale }/${ choice.helpLink.internalPage.slug }`"
+                    target="_blank"
+                    rel="noopener"
+                    class="tree__content-link"
```

Comment:
is that class used?

Reply:
Yes, it just does the same as the main link (put arrow in front of it, etc.)
I thought you might want to look at the styling, they need't be so similar.